src/response.ts

```diff
@@ -34,6 +34,14 @@ export const defaultResponse: Omit<MockedResponse, 'headers'> = {
   once: false,
 }
 
+const postProcessJSONBody = <T = Partial<MockedResponse>>(response: T): T => {
```

Comment:
Would also be a tranform I guess we just compose in. Let me know what you think.